cumulusci/utils/yaml/cumulusci_yml.py

```diff
@@ -211,6 +211,18 @@ class CumulusCIRoot(CCIDictModel):
     sources: Dict[str, Union[LocalFolderSourceModel, GitHubSourceModel]] = {}
     cli: CumulusCLIConfig = None
 
+    @classmethod
+    def schema(cls, *args, **kwargs):
+        json = super().schema(*args, **kwargs)
+        # Workaround VSCode-YAML vs pydantic.schema quirks
+        task_model = json["definitions"]["Task"]["properties"]
+        task_model["options"]["additionalProperties"] = True
+        task_model["ui_options"]["additionalProperties"] = True
```

Comment:
Note for later... Looking at the [docs](https://pydantic-docs.helpmanual.io/usage/schema/#field-customisation), it looks like we could also pass `additionalProperties` via `Field(..., additionalProperties=True)`.

Reply:
@jstvz Good idea!